code/backend/batch/utilities/search/AzureSearchHandler.py

```diff
@@ -1,13 +1,24 @@
+from typing import List
 from .SearchHandlerBase import SearchHandlerBase
+from ..helpers.LLMHelper import LLMHelper
 from ..helpers.AzureSearchHelper import AzureSearchHelper
 from ..common.SourceDocument import SourceDocument
 import json
+from azure.search.documents.models import VectorizedQuery
+import tiktoken
 
 
 class AzureSearchHandler(SearchHandlerBase):
+
+    _ENCODER_NAME = "cl100k_base"
+    _VECTOR_FIELD = "content_vector"
+
```

Comment:
thank you for having constants. i have noticed string literals being used in the code base, and it has been bothering me :)